.github/workflows/check-test-release.yml

```diff
@@ -91,12 +92,24 @@ jobs:
         wget https://raw.githubusercontent.com/Homebrew/homebrew-core/fb8323f2b170bd4ae97e1bac9bf3e2983af3fdb0/Formula/libomp.rb
         brew install ./libomp.rb
     - name: Install
-      if: steps.cache.pip-cache-dir.cache-hit != 'true'
       run: |
         pip install --upgrade pip setuptools wheel
         pip install pre-commit .[tests]
     - if: matrix.os == 'ubuntu-latest' && matrix.python == '3.10'
       run: pre-commit run pylint -a -v --show-diff-on-failure
+    - name: Run Heroku tests
```

Comment:
since this is run when heroku plugin changes, I think it's better to put it earlier in the order.